backend/app/views/spree/admin/roles/edit.html.erb

```diff
@@ -6,6 +6,8 @@
   <%= back_to_list_button(Spree::Role.model_name.human, admin_roles_path) %>
 <% end %>
 
+<%= render :partial => 'spree/admin/shared/error_messages', :locals => { :target => @role } %>
```

Comment:
@priyank-gupta this is @vinayvinsol commit so if I'll fix it now it's still will change the history, same as doing `rubocop -a` later on